data_engineering.qmd

```diff
@@ -1,121 +1,526 @@
 # Data Engineering
```

Comment:
Overall, really great, well laid out chapter! One very small overall comment: it would be nice to standardize the format of the subheadings in each section just to make a bit easier to find subsections when trying to refer back to earlier parts of the chapter. But again, mostly very small details in an overall great chapter.